TrackingTools/PatternTools/interface/Trajectory.h

```diff
@@ -357,6 +368,7 @@ class Trajectory
   float theChiSquaredBad=0;
 
   signed short theNumberOfFoundHits=0;
+  signed short theNumberOfFoundPixelHits=0;
```

Comment:
I think we will continue to call pixel the inner layers (BPIX and FPIX. FPIX most probably including the extension)
Phase2 OT will be called TOB and TID (do not ask why TOB and TID)
We assume we will Never need to mix run2 and phase2 geometries: we will disentagle at level of configuration